server/model/monitor.js

```diff
@@ -1508,9 +1508,9 @@ class Monitor extends BeanModel {
         return await R.getAll(`
             SELECT monitor_notification.monitor_id, monitor_notification.notification_id
             FROM monitor_notification
-            WHERE monitor_notification.monitor_id IN (?)
+            WHERE monitor_notification.monitor_id IN (${monitorIDs.map((_) => "?").join(",")})
         `, [
-            monitorIDs,
+            ...monitorIDs,
```

Comment:
Thank you for the fix, misunderstood it before.

Looks like the pass the array directly without the "merging syntax"   

```
R.getAll("SELECT...", monitorIDs);
```

Reply:
Oh, you're right.